src/main/java/de/hpi/bpt/argos/eventHandling/EventPlatformRestEndpointImpl.java

```diff
@@ -83,16 +87,17 @@ protected void loadDefaultEventTypes() {
 			// since the path is delivered as URI, it is represented as a HTML string. Thus we need to replace %20 with file system spaces.
 			File eventTypesDirectory = new File(eventTypesDirectoryPath.replaceAll("%20", " "));
 
-			for (File eventType : eventTypesDirectory.listFiles()) {
-				if (!eventType.getName().endsWith(".json")) {
+			for (File eventTypeFile : eventTypesDirectory.listFiles()) {
+				if (!eventTypeFile.getName().endsWith(".json")) {
 					continue;
 				}
 
-				loadDefaultEventType(eventType);
+				loadDefaultEventType(eventTypeFile);
 			}
 
 		} catch (NullPointerException e) {
-			logger.error("cannot find directory for default event types or not defined in properties", e);
+			logger.error("cannot find directory for default event types or not defined in properties");
+            logger.trace("Reason: ", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Reason: " 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)